.github/actions/regression-tests/action.yml

```diff
@@ -52,7 +52,7 @@ runs:
 
     - name: Run PyTests replication test
       id: second
-      if: ${{ inputs.run-only-on-ubuntu-latest == 'true' || (inputs.run-only-on-ubuntu-latest == 'false' && matrix.runner == 'ubuntu-latest') }}
+      if: ${{ (inputs.run-only-on-ubuntu-latest == 'true' && matrix.runner == 'ubuntu-latest') || (inputs.run-only-on-ubuntu-latest == 'false' }}
```

Comment:
why do we still need this if ? cant we just remove it?

Reply:
We can, I think we didn't want to run them on release. Let me check :)